### DISCUSSION TITLE
ImportError: cannot import name 'pyutils' from 'pyserini.index'

### DISCUSSION THREAD

**Discussion by XY2323819551** [castorini/pyserini#1341] [Q&A] [answered]
Hi, I want to convert internal docid to external docid, I find solution in [#33](https://github.com/castorini/pyserini/issues/33#issue-565057381). However, I met ImportError when I conducted "from pyserini.index import pyutils". So could you tell me why? Thank you.

**Comment by lintool** [comment, marked as answer]
Hi @XY2323819551 - the class in question has been refactored... try:

```python
>>> from pyserini.index.lucene import IndexReader
>>> reader = IndexReader.from_prebuilt_index('robust04')
...
```

**Reply by XY2323819551** [reply]
Thank you very much!
